DOCS/man/vo.rst

```diff
@@ -492,6 +492,13 @@ Available video output drivers are:
         Overlay index is zero based, and related to crtc.
         (default: 0)
 
+    ``--drm-aspect``
```

Comment:
is this option really needed? can't it just use 

1) `--monitoraspect`
2) if not set, then `--monitorpixelaspect`
3) if not set, then use kms connector values

If for some reason the previous behaviour is really wanted, you can always set `--monitorpixelaxpect=1`.